Fenced_Frames_Ads_Reporting.md

```diff
@@ -109,7 +109,7 @@ Note `window.fence` here is a new namespace for APIs that are only available fro
 
 ## registerAdBeacon
 
-A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. A beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent.
+A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. The registered URL's site is required to be attested for Protected Audience API. Please see [the Privacy Sandbox enrollment attestation model](https://github.com/privacysandbox/attestation#the-privacy-sandbox-enrollment-attestation-model). If the URL is attested for Protected Audience API, a beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent. Otherwise the beacon will not be sent. If the destination decides to redirect, it's that site's responsibility to share any data in the redirect URL as per the attestation guarantees. The redirected site is not checked for enrollment and attestation since the browser doesn't add any data directly to those redirects' URL.
```

Comment:
I'm having some trouble understanding "it's that site's responsibility to share any data in the redirect URL as per the attestation guarantees".  In particular, I'm wondering which site "that site" refers to and I'm wondering how attestation guarantees require sites to share data (I thought attestations were about privacy which is the opposite of sharing data)?

Reply:
I read "that site" as the redirecting site (the one that Chrome must verify is attested). 

And that the phrase "responsibility to share any data in the redirect URL" was more "should act in accord with its attestation w.r.t the API," or somesuch, if anything.

Why does the site (or sites, if multiple redirects) to which the attested site may redirect get a "free pass" (as in does not need to attest that)

>it will not use the Privacy Sandbox APIs or services for the purpose of learning that you are the same user across different sites or apps, and that it will not otherwise circumvent the privacy protections of the Privacy Sandbox

Reply:
For example, let's say the reporting beacon is sent to "A.com", then "A.com" redirects it to "B.com". Here "that site" refers to "A.com".

Since we only check the attestations of the initial beacon destination, in the case "A.com", we state that "A.com" should act in accordance with its attestation if it decides to share any data via the redirect url.

Note for the site the beacon is redirected to, in this case "B.com", the initial beacon's event data is removed. The request method is changed to a GET request. See [explainer here](https://github.com/WICG/turtledove/blob/main/Fenced_Frames_Ads_Reporting.md#redirects).

Reply:
@JensenPaul I have updated the explainer to clarify this, PTAL, thanks.